Shared/Sources/RevertItems.swift

```diff
@@ -13,16 +13,9 @@ enum RevertItems: String {
   case view = "ViewItems"
   case whatsNew = "NewItems"
 
-  var data: [[String: AnyObject]] {
-    guard let data = NSArray(contentsOf: self.URL) as? [[String: AnyObject]] else {
-      fatalError(self.invalidContentError)
-    }
-    return data
-  }
-
-  func newData<T: Decodable>() -> T {
+  func Data<T: Decodable>() -> T {
```

Comment:
Any reason for making a function name start with uppercased character?

Reply:
Oooh I missed this one. Yeah, no, please follow the Swift naming guidelines unless there's a really good reason not to.

Reply:
Ah my bad! This is related to the reason why I called `Foundation.Data()`. Unknowingly, I have changed the function name with uppercase prefix, so Xcode was complaining when I tried initiating Foundation's Data inside the function.

Thanks for pointing out!